submissions/sadoharu/html-css-popup/style.css

```diff
@@ -0,0 +1,298 @@
+}
+
+@font-face {
+    font-family: 'Roboto';
+    src: local('Roboto Bold'), local('Roboto-Bold'),
+        url('./fonts/Roboto-Bold.woff2') format('woff2'),
+        url('./fonts/Roboto-Bold.woff') format('woff');
+    font-weight: bold;
+    font-style: normal;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'icomoon';
+    src: url('./fonts/icomoon.woff2') format('woff2'),
+        url('./fonts/icomoon.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+    font-display: block;
+}
+
+[class^="icon-"], [class*=" icon-"] {
+  font-family: 'icomoon';
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+.icon-bell-solid:before {
+  content: "\e900";
+}
+
+.icon-grip-solid:before {
+  content: "\e901";
+  position: absolute;
+  transform: translateX(-8px);
+}
+
+*   {
+    font-family:"Roboto",sans-serif;
+    box-sizing: border-box;
+    text-decoration: none;
+}
+
+.header {
+    position: relative;
+    height: 100px;
+    background-image: linear-gradient(to bottom, #ebe7e200 , #d7caba);
+    display: flex;
+    border-radius: 10px;
+}
+
+.nav-bar {
+    position: absolute;
+    margin-top: 60px;
+    width: 100%;
+    height: 50px;
+    display: flex;
+    justify-content: flex-end;
+    padding: 5px;
+    border-bottom-left-radius: 10px;
+    border-bottom-right-radius: 10px;
+    box-shadow: 5px 5px 45px rgba(0, 0, 0, .15);
+    background-image: linear-gradient(to bottom, #e3dbce , #c0a584); 
+}
+
+.header .logo {
+    position: absolute;
+    padding: 10px;
+    display: flex;
+    align-items: center;
+    z-index: 1;
+    color: #613011;
+}
+
+.header .logo:focus{
+    outline: 1px solid #6130114f;
+    border-radius: 5px;
+}
+
+.header .logo img {
+    height: 100px;
+}
+
+.header .logo h1 {
+    font-weight: bold;
+    font-size: 20px;
+    margin-top: 60px;
+    transition: .3s all;
+}
+
+.header .logo h1:hover {
+    filter: drop-shadow(8px 8px 5px #6130114f);
+}
+
+.nav-bar .nav-bar_list {
+    list-style-type: none;
+    display: flex;
+    align-items: center;
+    padding-left: 0;
+    width: 220px;
+    justify-content: space-around;
+}
+
+.nav-bar .nav-bar_list_item {
+    padding-left: 0;
+    transition: .3s all;
+}
+
+.nav-bar .nav-bar_list_item:hover {
+    filter: drop-shadow(0 0 3px #613011b2);
+}
+
+.nav-bar .nav-bar_list_item a, .nav-bar .nav-bar_list_item label {
+    color: #613011;
+    font-size: 16px;
+    line-height: 20px;
+    cursor: pointer;
+}
+
+.nav-bar .nav-bar_list_item.popup-button {
+    position: relative;
+}
+
+.nav-bar .nav-bar_list_item a:focus {
+    outline: 1px solid #6130114f;
+    border-radius: 5px;
+}
+
+.nav-bar .nav-bar_list_item.popup-button .popup-icon:focus ~ .icon-grip-solid:before {
+    outline: 1px solid #6130114f;
+    border-radius: 5px;
+}
+
+.nav-bar .nav-bar_list_item.popup-button input{
```

Comment:
Avoid styling by HTML tag. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)